locales/el/el.json

```diff
@@ -1,618 +1,618 @@
-    "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.",
-    "Are you sure you want to detach the selected resources?": "Are you sure you want to detach the selected resources?",
-    "Are you sure you want to detach this resource?": "Are you sure you want to detach this resource?",
-    "Are you sure you want to force delete the selected resources?": "Are you sure you want to force delete the selected resources?",
-    "Are you sure you want to force delete this resource?": "Are you sure you want to force delete this resource?",
-    "Are you sure you want to restore the selected resources?": "Are you sure you want to restore the selected resources?",
-    "Are you sure you want to restore this resource?": "Are you sure you want to restore this resource?",
-    "Are you sure you want to run this action?": "Are you sure you want to run this action?",
-    "Are you sure you would like to delete this API token?": "Are you sure you would like to delete this API token?",
-    "Are you sure you would like to leave this team?": "Are you sure you would like to leave this team?",
-    "Are you sure you would like to remove this person from the team?": "Are you sure you would like to remove this person from the team?",
-    "Argentina": "Argentina",
-    "Armenia": "Armenia",
-    "Aruba": "Aruba",
-    "Attach": "Attach",
-    "Attach & Attach Another": "Attach & Attach Another",
-    "Attach :resource": "Attach :resource",
-    "August": "August",
-    "Australia": "Australia",
-    "Austria": "Austria",
-    "Azerbaijan": "Azerbaijan",
-    "Bahamas": "Bahamas",
-    "Bahrain": "Bahrain",
-    "Bangladesh": "Bangladesh",
-    "Barbados": "Barbados",
+    "American Samoa": "Αμερικανική Σαμόα",
+    "An error occured while uploading the file.": "Παρουσιάστηκε σφάλμα κατά τη μεταφόρτωση του αρχείου.",
+    "Andorra": "Ανδόρας",
+    "Angola": "Αγκόλα",
+    "Anguilla": "Ανγκουίλλα",
+    "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "Ένας άλλος χρήστης έχει ενημερώσει αυτόν τον πόρο από τότε που φορτώθηκε αυτή η σελίδα. Παρακαλώ ανανεώστε τη σελίδα και προσπαθήστε ξανά.",
+    "Antarctica": "Ανταρκτική",
+    "Antigua And Barbuda": "Αντίγκουα και Μπαρμπούντα",
+    "API Token": "Διακριτικό API",
+    "API Token Permissions": "Άδειες συμβόλων API",
+    "API Tokens": "Μάρκες API",
+    "API tokens allow third-party services to authenticate with our application on your behalf.": "Τα API tokens επιτρέπουν στις υπηρεσίες τρίτων να πιστοποιούν με την εφαρμογή μας για λογαριασμό σας.",
```

Comment:
When we have to use untranslated English words or terms within a Greek sentence, we shouldn't write them in plural form. "API tokens" should be just "API token", although I cant find a source confirming this, I just know it from "English" and "Greek" school.